content/posts/matplotlib/using-matplotlib-to-advocate-for-postdocs/index.md

```diff
@@ -36,7 +36,7 @@ I have used [Matplotlib](https://matplotlib.org) to plot the NIH stipend levels
 (y axis) for each year of postdoctoral experience (x axis) for the past 4 years
 of NIH guidelines (color). I have also looked at the inflation of years 2017--2019
 and increased the salaries of the previous year by that percentage (dashed lines).
-![Postdoc salary in the past 4 years.](gross_salary.png)
+![Postdoc salary in the past 4 years. ](gross_salary.png)
```

Comment:
```suggestion
![Plot of the NIH stipend level for postdoc versus the number of years of postdoc experience for every year from 2016 to 2018 inclusive. The x-axis ranges from 0 to 7 years of experience. The y-axis ranges from $45,000 USD to $60,000 USD. The plot also shows how these salaries would increase according to the rate of inflation during the same years. The overall message is that postdoc salaries are adjusted for inflation. See the blog post for additional information about the plot.](gross_salary.png)
```

Reply:
Nice work, @jarrodmillman! Because this chart is being used to illustrate the main point of the blog, it's critical summary. I only have a few pieces of feedback.
- I think the chart is `every year from 2016 to 2019` (instead of 2018)
- I don't think the ending `See the blog post…` sentence is needed, so personally I'd remove it. It's not wrong, though.
- I may add something to the message `The overall message is that postdoc salaries are adjusted for inflation nationally` since they make a point later to use rent in cities as a proxy for city-specific inflation.

Reply:
Done. Thanks!!